actions/actions.py

```diff
@@ -87,45 +52,32 @@ def name(self) -> Text:
     def run(self, dispatcher: CollectingDispatcher,
```

Comment:
missing docstring about what the function does.

Reply:
✔️